events/Organizations/InviteAccountToOrganization.json

```diff
@@ -9,6 +9,31 @@
     "mitreAttackTechniques": [
         "T1535 - Unused/Unsupported Cloud Regions"
     ],
+    "mitreAttackSubTechniques": [
+
+    ],
+    "unverifiedMitreAttackTechniques": [
+        {
+            "technique": "T1098 - Account Manipulation",
+            "reason": "Adding accounts to the organization can be used to manipulate account permissions and roles for persistence or escalation of privileges."
+        },
+        {
+            "technique": "T1071 - Application Layer Protocol",
+            "reason": "Using the API call to communicate and manage accounts in the organization can evade detection mechanisms focused on other layers."
+        },
+        {
+            "technique": "T1102 - Web Service",
+            "reason": "Utilizing AWS Organizations API as a web service to manage accounts can help in evading direct detection of unauthorized activities."
```

Comment:
Similar to Application Layer Protocol, all APIs can be executed from the AWS Console.

Reply:
Removed.